lib/vsc/filesystem/quota/tools.py

```diff
@@ -184,6 +184,7 @@ def process_user_quota(storage, gpfs, storage_name, filesystem, quota_map, user_
     """
     del filesystem
     del gpfs
+    del user_map
```

Comment:
where does this come from? does it work anywhere at all?

or can we populate it and reuse the lookups for all the filesystems?

Reply:
Hmm, good question. I guess we can do that.